### ISSUE TITLE
Class SenderShell could not be loaded

### ISSUE THREAD

**Issue by rajeevgandhis** [nodesagency/cakephp-email-queue#16]
Hi,
After successful configuration of cakephp-email-queue in my application. I am getting this error. I have the instructions for the manual method from the URL: https://github.com/nodesagency/cakephp-email-queue

If I run comment in,
/var/www/html/ABCD/app$ Console/cake EmailQueue.Sender -h
am getting ": No such file or directory" as an error message

If I run comment in,
/var/www/html/ABCD/app/Console$ cake EmailQueue.Sender -h
Am getting: "Error: Class SenderShell could not be loaded." as an error message

**Comment by ankr** [comment]
This repo seems no longer maintained.

I would suggest you try take a look here  https://github.com/FriendsOfCake/awesome-cakephp#email
Fx. lorenzo/cakephp-email-queue which is the original author of this plugin

**Comment by rajeevgandhis** [comment]
Yes, I had seen this but it refers version 3.x but I am using version 2.x. Is there any possibility?
